src/devtools/client/inspector/markup/components/Node.tsx

```diff
@@ -72,6 +73,7 @@ class _Node extends PureComponent<FinalNodeProps> {
       // Chrome sometimes ignores element.scrollIntoView() here,
       // calling it with a little delay fixes it
       setTimeout(() => el.scrollIntoView({ behavior: "smooth", block: "center" }));
+      _Node.timeOutID = setTimeout(() => el.scrollIntoView({ behavior: "smooth", block: "center" }));
```

Comment:
This change seems mistaken:
* We now have _two_ `setTimeout` calls where there was one before.
* Also the use of a static variable seems like a mistake, since setting and clearing the time out are both associated with instances of the component.